src/components/Modal/Modal.module.scss

```diff
@@ -4,16 +4,21 @@
   background: #141414;
   max-width: 940px;
   width: 50%;
+  max-height: 80%;
   z-index: 1;
   position: fixed;
   top: 20%;
+  overflow-y: auto;
```

Comment:
This fixed the scrollbar issue on the Modal in my browser 👍. There's another one on the &_overlay causing scrollbars to appear in the main window.